portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,18 @@
-#content {
+a {
```

Comment:
This will change the style of all your anchors. Do you want to scope it to just 

#about-content a

Reply:
added change